### PR TITLE
Fixes #RHIROS-1193 - Adding hash key while producing the kafka message

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

File: internal/services/recommender.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func ProcessEvent(msg *kafka.Message) {
 		if err != nil {
 			log.Errorf("Unable convert list_of_experiments to json: %s", err)
 		}
-		_ = p.SendMessage(msgBytes, &cfg.ExperimentsTopic)
+		_ = p.SendMessage(msgBytes, &cfg.ExperimentsTopic, kafkaMsg.Kafka_request_msg.Metadata.Org_id)
```
Comment (saltgen) [inline internal/services/recommender.go:108]: Shall we include the `request_id` as well along with the `org_id`, like so <org_id>-<request_id>?

Comment (patilsuraj767) [reply internal/services/recommender.go:108]: I think we should not add it considering a scenario when customer is in disconnected env and manually uploading 4 reports to consoledot in chronological order.

**Review by saltgen — APPROVED** [review]
Thanks @patilsuraj767, LGTM :+1:

**Comment by pgarciaq** [comment]
## IQE tests added for fleet-heatmap and oom-timeline

Added the following IQE test files in `iqe-ros-ocp-plugin` (branch `pgarciaq-rosocp-superpowers-phase15`):

### `test_fleet_heatmap.py` (GET /recommendations/openshift/fleet-heatmap)
- `test_fleet_heatmap_returns_200_with_correct_structure` — validates meta/data shape
- `test_fleet_heatmap_node_fields` — validates all node field types and valid values
- `test_fleet_heatmap_metric_cpu` / `test_fleet_heatmap_metric_memory` — metric parameter
- `test_fleet_heatmap_invalid_metric_returns_400` — bad metric rejected
- `test_fleet_heatmap_invalid_engine_returns_400` — bad engine rejected
- `test_fleet_heatmap_filter_by_cluster` — RBAC cluster filtering
- `test_fleet_heatmap_filter_cluster_no_match` — empty response for nonexistent cluster
- `test_fleet_heatmap_engine_cost` / `test_fleet_heatmap_engine_performance` — engine filter
- `test_fleet_heatmap_term_short` / `test_fleet_heatmap_term_long` — term filter with data_window
- `test_fleet_heatmap_defaults` — validates default metric/term/engine
- `test_fleet_heatmap_latest_update_format` — ISO timestamp validation
- `test_fleet_heatmap_machineset_and_instance_type` — string field presence

### `test_oom_timeline.py` (GET /recommendations/openshift/containers/{id}/oom-timeline)
- `test_oom_timeline_returns_200_with_correct_structure` — validates meta/data envelope
- `test_oom_timeline_data_entry_fields` — validates date/oom_kill_count types
- `test_oom_timeline_with_date_range` — start_date/end_date filtering
- `test_oom_timeline_invalid_date_returns_400` — bad date rejected
- `test_oom_timeline_start_after_end_returns_400` — invalid range rejected
- `test_oom_timeline_invalid_recommendation_id_returns_400` — non-UUID rejected
- `test_oom_timeline_nonexistent_id_returns_404` — unknown UUID returns 404
- `test_oom_timeline_count_matches_data_length` — meta.count == sum(oom_kill_count)
- `test_oom_timeline_dates_are_ascending` — ordering verification

Constants added to `iqe_ros_ocp/fixtures/constants.py`:
- `RECOMMENDATIONS_FLEET_HEATMAP`
- `RECOMMENDATIONS_OOM_TIMELINE`

**Pull request merged**